contributors/design-proposals/job.md

```diff
@@ -79,8 +105,21 @@ type JobSpec struct {
     // job should be run with. Defaults to 1.
     Completions *int
 
+    // Optional duration in seconds relative to the startTime that the job may be active
+    // before the system tries to terminate it; value must be positive integer
```

Comment:
- [ ] s/be positive integer/be a positive integer./  
- [ ] Did you consider using type `*uint` instead?  
  That still requires an admonishment about a nonzero value.